src/test/java/picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigarTest.java

```diff
@@ -54,75 +54,75 @@ protected UmiAwareMarkDuplicatesWithMateCigarTester getTester(final boolean allo
     private Object[][] testUmiSetsDataProvider() {
         return new Object[][] {{
                 // Test basic error correction using edit distance of 1
-                Arrays.asList(new String[] {"AAAA", "AAAA", "ATTA", "AAAA", "AAAT"}), // Observed UMI
-                Arrays.asList(new String[] {"AAAA", "AAAA", "ATTA", "AAAA", "AAAA"}), // Expected inferred UMI
-                Arrays.asList(new Boolean[] {false, true, false, true, true}), // Should it be marked as duplicate?
+                Arrays.asList("AAAA", "AAAA", "ATTA", "AAAA", "AAAT"), // Observed UMI
```

Comment:
I would revert these changes as they will collide with the same changes from @fleharty 's branch, and it doesn't look like the tests were changed here, unlike Mark's changes.